src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -1095,6 +1090,122 @@ public void Deconstruct(out int a, out int b)
         }
 
         [Fact]
+        [WorkItem(18629, "https://github.com/dotnet/roslyn/issues/18629")]
+        public void ValueTupleNotRequiredIfReturnIsNotUsed()
```

Comment:
I see that we already have tests covering constraints.

---
In reply to: [123337083](https://github.com/dotnet/roslyn/pull/20295#discussion_r123337083) [](ancestors = 123337083,123336564)